reconcile.py

```diff
@@ -78,6 +75,11 @@ def get_new_tracking_pos_costs_maps(
     ) for (k, v) in group_trackings_to_po.items()})
     po_to_cost_map.update(group_po_to_cost)
 
+  non_portal_reimbursements_retriever = NonPortalReimbursementsRetriever(config)
+  non_portal_reimbursements = non_portal_reimbursements_retriever.non_portal_reimbursements
```

Comment:
This is not really how you'd expect to get data from another class, or a retriever.  Maybe add a `get()` convenience method to that class that returns the `non_portal_reimbursements` field?  Or rename the class from retriever to something else?  Because it's not really being used as a retriever; the retrieving is happening in the initializer method itself.  So it's really more of just a "NonPortalReimbursements".

Reply:
yeah i think renaming is the way to go, and just separating it into the two separate dicts stored in two separate files 

smushing them together into the same object doesn't really gain us any benefits